### PR TITLE
T79384 feat: create EmptyStateCard component

### PR DESCRIPTION
**Pull request by OskiTheCoder**
## 📝 Changes

- Creates `EmptyStateCard` component

## ✅ Checklist

Easy UI has certain UX standards that must be met. In general, non-trivial changes should meet the following criteria:

- [x] Visuals match Design Specs in Figma
- [x] Stories accompany any component changes
- [x] Code is in accordance with our style guide
- [x] Design tokens are utilized
- [x] Unit tests accompany any component changes
- [x] TSDoc is written for any API surface area
- [x] Specs are up-to-date
- [x] Console is free from warnings
- [x] No accessibility violations are reported
- [x] Cross-browser check is performed (Chrome, Safari, Firefox)
- [x] Changeset is added

~Strikethrough~ any items that are not applicable to this pull request.

### CODE REVIEW

File: easy-ui-react/src/EmptyStateCard/EmptyStateCard.tsx [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+  const {
+    children,
+    gap = "2",
+    align = "space-between",
+    inlineAlign = "start",
+    ...restVerticalStackProps
+  } = props;
+
+  return (
+    <Card background="primary.800" borderRadius="lg" padding="5" boxShadow="1">
```
Comment (OskiTheCoder) [inline easy-ui-react/src/EmptyStateCard/EmptyStateCard.tsx:57]: checked with design about `boxShadow`; mockups have value that doesn't map to one of our card values.

File: easy-ui-react/src/EmptyStateCard/EmptyStateCard.tsx [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+   * Gap between `<EmptyStateCard.Header>` and `<EmptyStateCard.Body>`
+   * @default 2
+   */
+  primaryGap?: ResponsiveSpaceScale;
+  /**
+   * Gap between `<EmptyStateCard.Header>` and `<EmptyStateCard.Body>`
+   * block and `<EmptyStateCard.Action>`
+   * @default 2
+   */
+  secondaryGap?: ResponsiveSpaceScale;
```
Comment (OskiTheCoder) [inline easy-ui-react/src/EmptyStateCard/EmptyStateCard.tsx:25]: looking at the Figma, it looks as though we need to support a variation where the gap is different

Comment (kevinalexliu) [reply easy-ui-react/src/EmptyStateCard/EmptyStateCard.tsx:28]: Suggestion: what do you think of `textGap` for the gap between header and body and `blockGap` for the gap between cta and text?

**Review by kevinalexliu — DISMISSED** [review]
LGTM!

File: easy-ui-react/src/EmptyStateCard/EmptyStateCard.tsx (hunk 2 above)
Comment (OskiTheCoder) [reply easy-ui-react/src/EmptyStateCard/EmptyStateCard.tsx]: good call, i like that better

**Review by kevinalexliu — APPROVED** [review]

**Pull request merged**